areas-of-d-usage.dd

```diff
@@ -36,8 +36,6 @@ $(DIVC start-icons,
 )
 
 $(DIV,
-    <img id="dman" src="images/d3.png" border="0" align="left" alt="D Man">
```

Comment:
"Unfortunately" @CyberShadow is right here, this page is thought to be one of the first pages for professional D users. I added the D-man before I had the idea with the FontAwesome icons to make the page a bit more visually appealling.
(also does/did the D-man make the layout with the CSS-icons more tricky)